ext/standard/tests/general_functions/proc_open_sockets4.phpt

```diff
@@ -0,0 +1,69 @@
+--TEST--
```

Comment:
This is missing a `--SKIPIF--` section.

We generally use `PHP_OS_FAMILY !== 'Windows'` or this in other tests

Reply:
I left out SKIPIF because I want the test to verify that it pipes behave in exactly the same way on Linux and Windows systems when threads are used on Windows. The `threaded_pipes` option has to be ignored for all other builds.